docs/_static/js/banner.js

```diff
@@ -16,23 +16,24 @@ function hasCookie(name, value) {
 $(() => {
   const $wagtailspace = $(`
   <div class="wagtailspace">
-    <a href="https://us.wagtail.space/">
+    <a href="https://wagtail.space/2022/">
```

Comment:
This is extremely nitpicky, but adding the `www.` here would avoid a redirect :)

```suggestion
    <a href="https://www.wagtail.space/2022/">
```